packages/vscode-extension/src/utils/commonUtils.ts

```diff
@@ -134,3 +137,16 @@ export function anonymizeFilePaths(stack?: string): string {
 
   return updatedStack;
 }
+
+export async function isTeamsfx(): Promise<boolean> {
+  if (workspace.workspaceFolders && workspace.workspaceFolders.length > 0) {
+    const workspaceFolder = workspace.workspaceFolders[0];
+    const fxPath = await commonUtils.getProjectRoot(workspaceFolder.uri.fsPath, ".fx");
+    if (fxPath && fs.pathExistsSync(fxPath)) {
```

Comment:
better to use the async method to check if path exists?

Reply:
Fixed.